.gitlab-ci.yml

```diff
@@ -32,8 +32,10 @@ variables:
   S3_CP_OPTIONS: --only-show-errors --region us-east-1 --sse AES256
   S3_CP_CMD: aws s3 cp $S3_CP_OPTIONS
   S3_ARTEFACTS_URI: s3://dd-ci-artefacts-build-stable/$CI_PROJECT_NAME/$CI_PIPELINE_ID
+  S3_OMNIBUS_CACHE_BUCKET: dd-ci-datadog-agent-omnibus-cache-build-stable
   S3_DSD6_URI: s3://dsd6-staging/linux
   RELEASE_VERSION: nightly
+  BASE_ROLE_SESSION_NAME: $CI_PROJECT_NAME-$CI_PIPELINE_ID
```

Comment:
what's this var for? Don't see it used anywhere

Let's document what it does if it's useful :)